### PR TITLE
Revert mi test app docker to dotnet 7.0

### PR DESCRIPTION
**Pull request by hsubramanianaks**
This PR reverts the samples/managed-identity app docker file back to 7.0 and created a separate deploy-arm.yaml with correct MI client ID's so that this can be recreated easily. @cxznmhdcxz FYIP.

### CODE REVIEW

File: samples/managed-identity/.vscode/launch.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
             "OS-COMMENT5": "Enable launching a web browser when ASP.NET Core starts. For more information: https://aka.ms/VSCode-CS-LaunchJson-WebBrowser",
             "env": {
                 "ASPNETCORE_ENVIRONMENT": "Development",
-                "ASPNETCORE_URLS": "http://+:80"
+                "ASPNETCORE_URLS": "http://+:5000"
```
Comment (Vidya2606) [inline samples/managed-identity/.vscode/launch.json:23]: why are we changing the port here?

Comment (hsubramanianaks) [reply samples/managed-identity/.vscode/launch.json:23]: I shouldn't have committed this file, let me revert it. Thanks for catching this.

Comment (hsubramanianaks) [reply samples/managed-identity/.vscode/launch.json]: Reverted.

**Review by Vidya2606 — APPROVED** [review]
Looks good!

**Pull request merged**